exercises/simple-cipher/src/test/java/Step1RandomKeyCipherTest.java

```diff
@@ -1,13 +1,17 @@
+package java;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import java.util.regex.Pattern;
 
 /**
- * Step 1: Make a simple shift cipher
+ * Step 1: Random key cipher
```

Comment:
Like @FridaTveit said, can you remove all the comments? The names of the tests should explain them well enough.